mender/install_mender.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+set -e
+
+if [ "$EUID" -ne 0 ]
+  then echo "Please run with sudo"
+  exit
+fi
+
+
+
+function get_mender_connect()
+{
+   [ ! -d ./mender/mender-connect ] && git clone -b 2.0.1 https://github.com/mendersoftware/mender-connect.git ./mender/mender-connect
+   true
+}
+
+function install_mender_lib_from_package()
+{
+   apt install ./mender/libffi6_3.2.1-9_arm64.deb
+}
+
+function install_mender_client_from_package()
+{
+   cp -R ./mender/mender /etc/
+   cp -r ./mender/lib/mender /var/lib/
+   apt install ./mender/mender-client_3.1.0-1_arm64.deb
+}
+
+function install_mender()
+{
+   install_mender_lib_from_package
+   install_mender_client_from_package
+   install_mender_connect_from_sources
+} 
+
+
+function install_mender_connect_from_sources()
+{
+   install_go_compiler
+#   apt-get install libglib2.0-dev
+
+   echo "Building mender connect..."
+   cd ./mender/mender-connect
```

Comment:
Following the previous suggestions
https://github.com/perimeter-inc/system-dependencies/pull/4/files#r808020982

```suggestion
   cd ${SCRIPT_PATH}/mender/mender-connect
```